base/int.jl

```diff
@@ -132,12 +132,35 @@ fld(x::Unsigned, y::Signed) = div(x, y) - (signbit(y) & (rem(x, y) != 0))
 """
     mod(x, y)
 
-Modulus after flooring division, returning in the range ``[0,y)``, if `y` is positive, or
+Modulus after flooring division, returning a value in the range ``[0,y)``, if `y` is positive, or
 ``(y,0]`` if `y` is negative.
 
 ```julia
 x == fld(x,y)*y + mod(x,y)
 ```
+
+!!! note
+
+    If either `x` or `y` are floating point numbers, and they opposite signs, then the
```

Comment:
If we want to be strictly grammatically correct, this should be "If either `x` or `y` is a floating point number," since the subject is actually singular.